posthog/api/test_legacy/test_action_people.py

```diff
@@ -0,0 +1,529 @@
+from json import dumps as jdumps
```

Comment:
Is this copy-pasta from an existing test new code?

Reply:
Yep, all the `test_legacy/*` tests are straight-up copied and pasted (only paths for `posthog.` imports adjusted).

Reply:
Yep as in: it's copypasta.